src/TicketNow/TicketNow.Api/appsettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "ConnectionStrings": {
-    "local": "Server=localhost;Database=ticket_now;User ID=sa;Password=1q2w3e4r@#$;Trusted_Connection=false;TrustServerCertificate=true;" //add to azure key vault
+    "local": "Server=localhost\\SQLEXPRESS01;Database=ticket_now;Trusted_Connection=True;TrustServerCertificate=True" //add to azure key vault
```

Comment:
aqui tb

Reply:
`{
  "ConnectionStrings": {
    "local": "Server=localhost;Database=ticket_now;User ID=sa;Password=1q2w3e4r@#$;Trusted_Connection=false;TrustServerCertificate=true;" //add to azure key vault
  },
`